web_external/js/models/WmsSourceModel.js

```diff
@@ -0,0 +1,23 @@
+minerva.models.WmsSourceModel = minerva.models.SourceModel.extend({
+
+    createSource: function (name, baseURL, projection) {
+        var params = {
+            name: name,
+            baseURL: baseURL,
+            projection: projection
```

Comment:
It's possible for a WMS server (and its layers) to support more than one projection.  Though for displaying on top of common basemaps like OSM, Google, etc, web mercator (EPSSG:900913/EPSG:3857) is likely the only one that matters.